src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts

```diff
@@ -96,6 +100,12 @@ class AccessibilityHelpProvider implements IAccessibleContentProvider {
 			}
 		}
 
+		if (this._contextKeyService.getContextKeyValue<boolean>(CommentContextKeys.activeEditorHasCommentingRange.key)) {
```

Comment:
the context key is undefined here 🤔 @alexr00

Reply:
I can see the same. This is strange. The key is done the same way as `activeCursorHasCommentingRange`, and that one works as expected (when your cursor is in a commenting range the command palette has the "Add Comment" command). I'll keep looking into this.

Reply:
Talked to Joao and he reminded me that each context key service is tied to a dom node. I am setting the context in the editors context.

```suggestion
		const editorContext = this._contextKeyService.getContext(this._editor.getDomNode()!);
		if (editorContext.getValue<boolean>(CommentContextKeys.activeEditorHasCommentingRange.key)) {
```